scripts/bootstrap

```diff
@@ -345,35 +399,35 @@ case "${what}" in
         ;;
 
     conf*)
-        local_action=true
-        header_action=true
-        tool_action=true
-        make_action=true
+        local_action="${TRUE}"
+        header_action="${TRUE}"
+        tool_action="${TRUE}"
+        make_action="${TRUE}"
         ;;
 
     make*)
-        local_action=true
-        header_action=true
-        config_action=true
+        local_action="${TRUE}"
+        header_action="${TRUE}"
+        config_action="${TRUE}"
         ;;
 
     none)
-        local_action=true
-        header_action=true
-        tool_action=true
-        make_action=true
-        config_action=true
+        local_action="${TRUE}"
+        header_action="${TRUE}"
+        tool_action="${TRUE}"
+        make_action="${TRUE}"
+        config_action="${TRUE}"
         ;;
 
 esac
 
 # Bootstrap the package.
 
-if [ -n "${verbose}" ]; then
-    echo "${local_action} && ${tool_action} && ${header_action} && ${make_action} && ${config_action}"
-fi
-
-${local_action} && ${tool_action} && ${header_action} && ${make_action} && ${config_action}
+try_tool_action "${local_action}"
+try_tool_action "${tool_action}"
+try_tool_action "${header_action}"
+try_tool_action "${make_action}"
+try_tool_action "${config_action}"
```

Comment:
now I see why exit() would be preferable for you

Reply:
does this file pass shellcheck?

Reply:
I also incorporated the changes from your PR to eliminate the unused variables.